setup.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python

```

Comment:
Got the following error 
```

/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running install
running build
running install_egg_info
Writing /usr/local/lib/python3.5/dist-packages/OMSens-1.0.0.egg-info
error: [Errno 13] Permission denied: '/usr/local/lib/python3.5/dist-packages/OMSens-1.0.0.egg-info'
````

I solved this by installing numpy, maybe it was already installed for your Python3 
With sudo I got 
```
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running install
running build
running install_egg_info
Writing /usr/local/lib/python3.5/dist-packages/OMSens-1.0.0.egg-info
Rut.for:760:72: Warning: Deleted feature: PAUSE statement at (1)
Rut.for:764:72: Warning: Deleted feature: PAUSE statement at (1)
Rut.for:788:72: Warning: Deleted feature: PAUSE statement at (1)
Rut.for:1073:72: Warning: Deleted feature: ASSIGN statement at (1)
Rut.for:1077:16:

 20    go to next, (30,40,70,80)
                1
Warning: Deleted feature: Assigned GOTO statement at (1)
Rut.for:1079:72: Warning: Deleted feature: ASSIGN statement at (1)
Rut.for:1084:72: Warning: Deleted feature: ASSIGN statement at (1)
Rut.for:1087:72: Warning: Deleted feature: ASSIGN statement at (1)
CURVI files compiled.
Traceback (most recent call last):
  File "setup.py", line 104, in <module>
    f2py_call = call([f2py_path, "-c", "-I.", "Curvif.o", "Rutf.o", "Rut.o", "-m", "curvif_simplified", "curvif_simplified.pyf", "Curvif_simplified.f90"], cwd="fortran_interface")
  File "/usr/lib/python3.5/subprocess.py", line 557, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.5/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'f2py'
```

Reply:
`numpy` should be installed as part of `pandas`. The real problem seems to be, 

```
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
```

Reply:
Maybe you first need todo `pip install setuptools`

Reply:
Will try it out for me the script started to wrok after I installed numpy, still I got those messages for that case as well. I am still rebuilding OMC.. (I had some other unrelated compilation issues)

Reply:
I will check it out tomorrow and or later this evening as well. Hopefully I will have OpenModelica up and running until then